docs/topics/db/aggregation.txt

```diff
@@ -339,6 +349,26 @@ you can issue the query::
 This query generates an annotated result set, and then generates a filter
 based upon that annotation.
 
+If you need two annotations with two separate filters you can use
+the ``filter`` argument with any aggregate. For example, to generate a list of
+authors with a count of highly rated books you can issue the query::
+
+    >>> highly_rated = Count('books', filter=Q(books__rating__gte=7)
+    >>> Author.objects.annotate(num_books=Count('books'), highly_rated_books=highly_rated))
+
+Each Author object in the annotated result set will have a ``num_books``
+as well as a ``highly_rated_books`` attribute.
+
+.. note::
```

Comment:
Please add a blank line in-between. The line limit is 79 for documentation, so you can easily squeeze the paragraph a little bit.